### DISCUSSION TITLE
How about adding an option to support Vim-like line-based yank/paste.

### DISCUSSION THREAD

**Discussion by DogLooksGood** [meow-edit/meow#239] [Ideas]
In Vim, if you copy a line, for example, by `yy`. Then paste will insert a whole line.  In Meow, when selecting a line, the newline will not be included. But when you do save or kill, the newline will be included.

The downsides are:
1. Meow manipulates the copy content by adding a newline.
2. Before paste, usually we have to put the cursor at the beginning of the line.  

How about we add an option to do line-based insertion for line copy/cut. We use a special flag or something like that to indicate we are doing line copy.

**Comment by gf3** [comment]
I would find this useful!

I find that typically when pasting I am moving blocks of code or lines around, rather than surgically pasting within a single line.
